app/code/pagetype/HomePage.php

```diff
@@ -9,6 +9,12 @@
 
 class HomePage extends Page
 {
+    /**
+     * Database name
+     *
+     * @var string
+     */
+    private static $table_name = "Home page";
```

Comment:
ideally we would want this to not hhave spaces and keep it in caps

i.e. `HomePage` :)